src/test/test-cgroup-setup.c

```diff
@@ -16,10 +16,8 @@ static void test_is_wanted_print_one(bool header) {
         log_info("-- %s --", __func__);
         assert_se(proc_cmdline(&cmdline) >= 0);
         log_info("cmdline: %s", cmdline);
-        if (header) {
-                log_info("default-hierarchy=" DEFAULT_HIERARCHY_NAME);
```

Comment:
Will the unit test fail if running on a legacy hierarchy? Build systems could run on ancient kernels and so on

Reply:
This only tests whether `cg_is_unified_wanted` and friends return the correct result with different kernel cmdline. I don't it will be affected.

Reply:
Ok sounds good then